tests/PHPStan/Analyser/data/integer-range-types.php

```diff
@@ -254,16 +255,18 @@ public function math($i, $j, $z, $pi, $r1, $r2, $rMin, $rMax, $x, $y) {
 		assertType('float|int<1, max>', $rMax / 4);
 
 		assertType('int<6, 20>', $r1 + $r2);
-		assertType('int<-4, 0>', $r1 - $r2);
+		assertType('int<-9, 5>', $r1 - $r2);
 		assertType('int<5, 100>', $r1 * $r2);
 		assertType('float|int<0, 1>', $r1 / $r2);
 
+		assertType('int<-99, 19>', $r1 - $r3);
```

Comment:
this line is covering https://phpstan.org/r/0f2bf43b-e5a6-436b-ac44-e62a2c60ae94 from  "Wrong inference with Minus" from https://github.com/phpstan/phpstan/issues/5614